content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -205,8 +205,10 @@ by the Kubernetes cluster, and then present that certificate to the Kubernetes A
 ### Create private key
 
 The following scripts show how to generate PKI private key and CSR. It is
-important to set CN and O attribute of the CSR. CN is the name of the user and
-O is the group that this user will belong to. You can refer to
+important to set the CN of the CSR. CN is the name of the user. O should be set
+as the already defined group that this user will belong to,  if such a group exists.
```

Comment:
nit.
```suggestion
as the already defined group that this user will belong to, if such a group exists.
```

Reply:
> already defined group

specifying an O entry in a client certificate is all that is required to "define" the group